packages/web-client/src/helpers/share/functionsNG.ts

```diff
@@ -81,12 +81,12 @@ export function buildIncomingShareResource({
     fileId: driveItem.remoteItem.id,
     storageId,
     parentFolderId: driveItem.parentReference?.id,
-    sdate: driveItem.remoteItem.shared.sharedDateTime,
+    sdate: driveItem.remoteItem.lastModifiedDateTime,
     indicators: [],
     tags: [],
     webDavPath: buildWebDavSpacesPath([SHARE_JAIL_ID, id].join('!'), '/'),
     sharedBy: driveItem.remoteItem.createdBy.user,
-    owner: driveItem.remoteItem.shared.owner.user,
+    owner: driveItem.remoteItem.createdBy.user,
```

Comment:
`/sharedWithMe` doesn't provide `remoteItem.shared` property. was removed in https://github.com/owncloud/ocis/pull/8322
>- the 'shared' property is no longer available (the relevant information from that
moved to the 'invitation' property of the individual permissions.

There is no following data in the response:
```json
"shared": {
  "owner": {
    "user": {
      "displayName": "Admin",
      "id": "d75bb461-d981-42cd-a072-ba00a9213e98"
    }
  },
  "sharedBy": {
    "user": {
      "displayName": "Admin",
      "id": "d75bb461-d981-42cd-a072-ba00a9213e98"
    }
  },
  "sharedDateTime": "2024-02-07T06:52:49.802308497Z"
}
```
Because of that share-with-me page doesn't show the list of shares.
We removed the use of `Shared` and that fixes the shares list in shared-with-me page. But I should not be doing so because there should be other things to clean up along with it.

@JanAckermann @JammingBen Could you guys look at it?

Reply:
There is already a PR for this: https://github.com/owncloud/web/pull/10430

I'll check in with the rest of the team so we can get this merged.

Reply:
Ahh, That's great. Thanks :smiley: 
I will close this one in favor of #10430